src/components/Asset/AssetContent/index.tsx

```diff
@@ -78,6 +79,11 @@ export default function AssetContent({
               </Button>
             </div>
           )}
+          <RelatedAssets
+            tags={asset?.metadata?.tags}
+            dtAddress={asset?.datatokens[0]?.address}
```

Comment:
if you actually intended to make sure this does not break even if `asset.datatokens` does not exist, then it needs to be `asset?.datatokens?.[0]?.address`